02-wrangling.Rmd

```diff
@@ -175,7 +188,7 @@ lang_mother_tidy
 > *to read clearly. When doing this, it is important to end the line with a comma `,` so that R*
 > *knows the function should continue to the next line.*
 
-The data above is now tidy because all 3 criteria for tidy data have now been met:
+The data above is now tidy because all three criteria for tidy data have now been met:
```

Comment:
In the "3.4.4 Going from long to wide using pivot_wider" section below, I think the second sentence needs to be a general example of what an observation split across two rows might look like.

Reply:
Made an example with a graphic